### DISCUSSION TITLE
border_color not working

### DISCUSSION THREAD

**Discussion by DMS055** [anuraghazra/github-readme-stats#1865] [Q&A] [answered]
For some reason I couldn't get the border to change colors (stats card)

**Expected behaviour**
Border should change colors as it's not disabled

**Additional context**
When using it with lang card it seems to work without an issue

**Comment by rickstaa** [comment, marked as answer]
@DMS055 thanks for your question. The following syntax works for me:

```markdown
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&bg_color=0000&title_color=ffff&text_color=ff00ff&icon_color=00ff00&border_color=0000ff)](https://github.com/anuraghazra/github-readme-stats)
```

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&bg_color=0000&title_color=ffff&text_color=ff00ff&icon_color=00ff00&border_color=0000ff)](https://github.com/anuraghazra/github-readme-stats)

I think the hex needs to have a minimum length of 4.

**Reply by DMS055** [reply]
Thanks for an answer.
I will try it as soon as I get home and I'll update you on the result

**Reply by DMS055** [reply]
@rickstaa managed to fix it!
Turns out I was using some other (probably older) version of the main URL but I found and replaced it eventually and it works now.
Anyway thanks for help!
